### PR TITLE
Update Dockerfile.dapper and .drone.yml

### PR DESCRIPTION
**Pull request by imikushin**
Update Dockerfile.dapper and .drone.yml

### CODE REVIEW

**Comment by nrvale0** [comment]
.drone.yml needs the following due to dind:

``` YAML
  privileged: true
```

@imikushin

**Comment by nrvale0** [comment]
I don't have the necessary perms on the repo but if someone could go here:

https://drone.rancher.io/account/rancher

and both enable the repo's build and click the gear to flip the "trusted" option on that would be excellent.

@imikushin @cjellick

**Comment by imikushin** [comment]
Looks like it's working with drone-0.5 now.

File: .drone.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
-image: rancher/dind:v1.10.0-rancher1
-script:
- - wrapdocker
- - make ci
+#workspace:
```
Comment (cjellick) [inline .drone.yml]: Is there a reason for having this in here but commented out?

Comment (imikushin) [inline .drone.yml]: Removing it: not needed anymore.

**Comment by imikushin** [comment]
Updated

**Comment by cjellick** [comment]
LGTM

**Pull request merged**